util/winutil/authenticode/mksyscall.go

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package authenticode
+
+//go:generate go run golang.org/x/sys/windows/mkwinsyscall -output zsyscall_windows.go mksyscall.go
+//go:generate go run golang.org/x/tools/cmd/goimports -w zsyscall_windows.go
+
+// The following two declarations use the '?' suffix because Windows 7.
```

Comment:
Because Windows 7 what?

Reply:
LOL! Actually I can remove that because we've now dropped Win7 support.